coral-component-table/src/scripts/Table.js

```diff
@@ -1570,6 +1576,21 @@ class Table extends BaseComponent(HTMLTableElement) {
       headerCell.setAttribute('sortabledirection', column.sortableDirection);
       headerCell.setAttribute('aria-sort',
         column.sortableDirection === sortableDirection.DEFAULT ? 'none' : column.sortableDirection);
+
+      let liveRegion = this._elements.liveRegion;
+      if(liveRegion) {
+        if(column.sortableDirection === sortableDirection.DEFAULT) {
+          liveRegion.innerText = '';
+        } 
+        else {
+          if(headerCell.content.textContent.trim().length) {
+            liveRegion.innerText = i18n.get('sorted by column {0} in ' + column.sortableDirection + ' order', headerCell.content.textContent);
+            setTimeout(function() {
```

Comment:
why do you use setTimeout here ?

Reply:
setTimeout is used to again set 'aria-live' to 'off' because it has been announced.

Reply:
but why `2500` and not `100` for example ?

Reply:
After removing timeout, voiceover never announces live Region content change. So timeout is required.

Reply:
Added requestAnimationFrame

Reply:
> but why `2500` and not `100` for example ?
> or can you use requestAnimationFrame ? or remove the setTimeout ?

A screen reader needs time to pick up on the content change and time to announce the string. Using requestAnimationFrame, will mean that the live region will never get announced, because the content will change, but the live region will be turned off with the next render frame.